src/gdal_layer.cpp

```diff
@@ -411,7 +411,7 @@ NAN_GETTER(Layer::dsGetter) {
  * @name srs
  * @instance
  * @memberof Layer
- * @type {SpatialReference}
+ * @type {SpatialReference|null}
```

Comment:
You will have to change the affected tests to this form:

```ts
const geom = feature.getGeometry()
assert.isNotNull(geom)
assert.isFalse(geom.isEmpty())
```

Reply:
added missing  asserts for Feature.getGeometry(), Layer.srs, MDArray.srs and RasterBand.getMaskBand()